client/src/components/ProfilePhoto/useStyles.ts

```diff
@@ -0,0 +1,60 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(() => ({
+  root: {
+    marginTop: '10rem',
+    marginBottom: '10rem',
+    alignItems: 'center',
+    justifyContent: 'center',
+    minWidth: '20rem',
+  },
+  profileContainer: {
+    alignItems: 'center',
+    minHeight: '60vh',
+    margin: 'auto',
+  },
+  contentWrapper: {
+    display: 'flex',
+    justifyContent: 'center',
+  },
+  title: {
+    marginTop: '5rem',
+  },
+  titleText: {
+    fontFamily: 'Roboto',
+    fontWeight: 'bolder',
+    fontSize: '1.5rem',
+    margin: 'auto',
+  },
+  avatar: {
+    marginTop: '4rem',
+    width: 150,
+    height: 150,
+  },
+  caption: {
+    marginTop: '2rem',
+    color: '#9e9e9e',
+    width: '10rem',
+    textAlign: 'center',
+  },
+  captionText: {
+    fontSize: '12px',
```

Comment:
Similarly here, we should try to avoid manually adjusting font size. Have you tried the `caption` variant?